app/data/repository-list.txt

```diff
@@ -12,3 +12,4 @@ cerc-io/laconic-cns-cli
 cerc-io/mobymask-watcher
 cerc-io/watcher-ts
 vulcanize/assemblyscript
+vulcanize/eth_probe
```

Comment:
Can we change the repo name to use a dash?

Reply:
Sounds good to me, but I don't have the necessary permissions.  Whom should I ask?